source/FreeRTOS_ND.c

```diff
@@ -1057,20 +1058,6 @@
                    {
                        size_t uxICMPSize;
                        BaseType_t xCompare;
-                       NetworkEndPoint_t * pxEndPointFound = FreeRTOS_FindEndPointOnIP_IPv6( &( pxICMPHeader_IPv6->xIPv6Address ) );
```

Comment:
It is always nice to remove code, isn't it?
This is a good cleanup: the original `pxEndPoint` should be used to reply to the sender.